### ISSUE TITLE
Users should be able to set a display name instead of using email.

### ISSUE THREAD

**Issue by creisman** [Full-House-UW/fullhouse#143]
It won't be pretty if everyone is referred to by their email. I think it should be added into user settings to allow the user to use a display name. If it's blank, then we use email.

**Comment by natmote** [comment]
I believe there's a field for first and last name in the django auth models. Should we just use that?

**Comment by j3ffml** [comment]
depends on #188

**Comment by snuriel92** [comment]
I have not changed the greeting to use the user's name (rather than email) because we need to add that to user settings and the model. I tried to add it because I thought that someone said that user's already have a first_name field when created in django and so I tried to use that but it didn't work, as far as I could tell. I didn't want to try adding anything to the model because I didn't want to mess anything up/don't know shit about migration or adding columns or whatever needs to be done.

Nat, I'm just assigning you to this so we can talk about what we need to do but feel free to re-assign to someone else
